graphdatascience/tests/unit/test_init.py

```diff
@@ -9,7 +13,13 @@
 
 @pytest.mark.parametrize("major, minor, patch, version_string", GDS_INIT_VERSION_TESTDATA)
 def test_gds_init_version(major: int, minor: int, patch: int, version_string: str) -> None:
-    gds = GraphDataScience(CollectingQueryRunner(ServerVersion.from_string(version_string)), arrow=False)
+    arrow_info = ArrowInfo(listenAddress="foo.bar", enabled=True, running=True, versions=[])
+    query_runner = CollectingQueryRunner(
+        ServerVersion.from_string(version_string),
+        result_mock=DataFrame([asdict(arrow_info)]),
+    )
+
+    gds = GraphDataScience(endpoint=query_runner, arrow=False)
```

Comment:
interesting setup. arrow is up and online, but client refuses it. it is a valid configuration.